src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -139,6 +147,8 @@ public override void SetResponse(BlittableJsonReaderObject response, bool fromCa
         private bool _first = true;
         private long _operationId = -1;
 
+        public CompressionLevel UseGzipCompressionLevel = CompressionLevel.Fastest;
```

Comment:
Maybe rename property to 'CompressionLevel' ? 'UseGzipCompressionLevel' does not sound good.